src/display-buffer.coffee

```diff
@@ -465,6 +465,10 @@ class DisplayBuffer extends Model
 
     folds
 
+  # Returns all the folds that contain the given row range.
+  foldsContainingBufferRowRange: (startRow, endRow) ->
```

Comment:
Makes sense, but in that case, if a fold intersects a single row it also contains it. In this case it's a range of rows, and we know these folds don't fully contain the range, otherwise there would only be one big fold. I'd be down renaming the other method for clarity/consistent if it isn't being used by packages (it's private but still might as well avoid breakages just in case).